### DISCUSSION TITLE
Unable to add GitHub users into an Organization in GitHub Enterprise with Evaluation License

### DISCUSSION THREAD

**Discussion by gbidsilva** [community/community#22731] [Enterprise]
<p>I have set up GitHub Enterprise server in one of my VMs using the VMWare image and it is successfully up and running.<br>
During the GitHub setup I have created one admin user and with that user I was able to set up a new organization (‘orgA’) and a repository.<br>
After this I have created a new  GitHub user (‘userA’) using GitHub server management page : stafftools/users/invite<br>
Creation of new user was successful and I was able to login with created new user credentials.</p>
<p>However, I am not able to add newly created user (‘userA’) into my organization (‘orgA’). It simply does not appear in the filter box.</p>
<p>Is this something to do with the evaluation license or am I doing something wrong?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/gbidsilva">@gbidsilva</a> - Thank you for being here and for trying GitHub Enterprise. There should not be a limitation because of a trial status. Are you able to restart your VM and see if this triggers a reindex?</p>

**Comment by gbidsilva** [comment]
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a><br>
Once I restarted the VM, this issue got resolved.<br>
After restarting, newly creating users are instantly available for organization additions.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Awesome, glad you got it going. We’re looking forward to seeing you around!</p>
